x-pack/plugins/serverless_search/public/application/constants.ts

```diff
@@ -12,5 +12,16 @@ export const INDEX_NAME_PLACEHOLDER = 'index_name';
 
 // Paths
 export const BASE_CONNECTORS_PATH = 'connectors';
+export const BASE_WEB_CRAWLERS_PATH = 'web_crawlers';
 export const EDIT_CONNECTOR_PATH = `${BASE_CONNECTORS_PATH}/:id`;
+export const ELASTIC_MANAGED_CONNECTOR_PATH = '/elastic_managed';
```

Comment:
Better to construct these the way the edit connector path above is constructed

Reply:
If I modify these constants in this:
```
export const ELASTIC_MANAGED_CONNECTOR_PATH = `${BASE_CONNECTORS_PATH}/elastic_managed`;
export const ELASTIC_MANAGED_WEB_CRAWLERS_PATH = `${BASE_WEB_CRAWLERS_PATH}/elastic_managed`;
```

Consuming them from the connectors_router.tsx like: 

```
<Route exact path={ELASTIC_MANAGED_CONNECTOR_PATH}>
  <ConnectorsElasticManaged />
</Route>
```

Generates a non working route URL:

`http://localhost:5601/app/connectors/connectors/elastic_managed`

With a duplicated `connectors/connectors...
`